SQL/Archive/2018-05-28_publication_schema.sql

```diff
@@ -0,0 +1,160 @@
+  CONSTRAINT `UK_publication_collaborator_1` UNIQUE (PublicationID, PublicationCollaboratorID),
+  CONSTRAINT `FK_publication_collaborator_1` FOREIGN KEY(`PublicationID`) REFERENCES `publication` (`PublicationID`),
+  CONSTRAINT `FK_publication_collaborator_2` FOREIGN KEY(`PublicationCollaboratorID`) REFERENCES `publication_collaborator` (`PublicationCollaboratorID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+-- Publication - Variable of Interest  relational table
+DROP TABLE IF EXISTS publication_parameter_type_rel;
+CREATE TABLE `publication_parameter_type_rel` (
+    `PublicationID` int(10) unsigned NOT NULL,
+    `ParameterTypeID` int(10) unsigned NOT NULL,
+    CONSTRAINT `UK_publication_parameter_type_1` UNIQUE (PublicationID, ParameterTypeID),
+    CONSTRAINT `FK_publication_parameter_type_rel_1` FOREIGN KEY (`PublicationID`) REFERENCES `publication` (`PublicationID`),
+    CONSTRAINT `FK_publication_parameter_type_rel_2` FOREIGN KEY (`ParameterTypeID`) REFERENCES `parameter_type` (`ParameterTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+DROP TABLE IF EXISTS publication_test_names_rel;
+CREATE TABLE `publication_test_names_rel` (
+    `PublicationID` int(10) unsigned NOT NULL,
+    `TestNameID` int(10) unsigned NOT NULL,
+    CONSTRAINT `FK_publication_test_names_rel_1` FOREIGN KEY (`PublicationID`) REFERENCES `publication` (`PublicationID`),
+    CONSTRAINT `FK_publication_test_names_rel_2` FOREIGN KEY (`TestNameID`) REFERENCES `test_names` (`ID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+-- Publication Uploads
+DROP TABLE IF EXISTS publication_upload_type;
+CREATE TABLE `publication_upload_type` (
+  `PublicationUploadTypeID` int(2) unsigned NOT NULL,
+  `Label` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+DROP TABLE IF EXISTS publication_upload;
+CREATE TABLE `publication_upload` (
+    `PublicationUploadID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `PublicationID` int(10) unsigned NOT NULL,
+    `PublicationUploadTypeID` int(2) unsigned NOT NULL,
+    `URL` varchar(255) NOT NULL,
+    `Version` varchar(255),
+    `Citation` text,
+    PRIMARY KEY (`PublicationUploadID`),
+    CONSTRAINT `UK_publication_upload_1` UNIQUE (URL),
+    CONSTRAINT `FK_publication_upload_1` FOREIGN KEY (`PublicationID`) REFERENCES `publication` (`PublicationID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+DROP TABLE IF EXISTS publication_users_edit_perm_rel;
+CREATE TABLE `publication_users_edit_perm_rel` (
+  `PublicationID` int(10) unsigned NOT NULL,
+  `UserID` int(10) unsigned NOT NULL,
+  CONSTRAINT `FK_publication_users_edit_perm_rel_1` FOREIGN KEY (`PublicationID`) REFERENCES `publication` (`PublicationID`),
+  CONSTRAINT `FK_publication_users_edit_perm_rel_2` FOREIGN KEY (`UserID`) REFERENCES `users` (`ID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+DELETE FROM publication_upload_type;
+INSERT INTO publication_upload_type VALUES (1, 'Paper');
```

Comment:
This, and the following inserts, should be moved bellow the table definition.
Also, the first value should be created by the auto_increment mechanism